airbyte-ci/connectors/metadata_service/lib/metadata_service/commands.py

```diff
@@ -24,7 +24,28 @@ def log_metadata_upload_info(metadata_upload_info: MetadataUploadInfo):
         )
     if metadata_upload_info.icon_uploaded:
         click.secho(
-            f"The icon file {metadata_upload_info.metadata_file_path} was uploaded to {metadata_upload_info.icon_blob_id}.", color="green"
+            f"The icon file for {metadata_upload_info.metadata_file_path} was uploaded to {metadata_upload_info.icon_blob_id}.",
+            color="green"
+        )
+    if metadata_upload_info.doc_version_uploaded:
+        click.secho(
+            f"The doc file for {metadata_upload_info.metadata_file_path} was uploaded to {metadata_upload_info.doc_version_blob_id}.",
+            color="green"
+        )
+    if metadata_upload_info.doc_latest_uploaded:
+        click.secho(
+            f"The doc file for {metadata_upload_info.metadata_file_path} was uploaded to {metadata_upload_info.doc_latest_blob_id}.",
+            color="green"
+        )
+    if metadata_upload_info.doc_inapp_version_uploaded:
+        click.secho(
+            f"The inapp doc file for {metadata_upload_info.metadata_file_path} was uploaded to {metadata_upload_info.doc_inapp_version_blob_id}.",
+            color="green"
+        )
+    if metadata_upload_info.doc_inapp_latest_uploaded:
+        click.secho(
```

Comment:
super nit: We're getting a little bit long in the ifs here

I know I started it 😅

But if we created a helper I would not be mad

```py
def log_if_uploaded(metadata_upload_info: MetadataUploadInfo, upload_success_key: str, upload_id_key: str, name: str):
    success = metadata_upload_info[upload_success_key]
    if success:
        path = metadata_upload_info[upload_id_key]
        click.secho(
            f"The {name} file for {metadata_upload_info.metadata_file_path} was uploaded to {path}.",
            color="green"
        )

log_if_uploaded(metadata_upload_info, "icon_uploaded", "icon_blob_id", "icon")
```